dhis-2/dhis-services/dhis-service-program-rule/src/main/java/org/hisp/dhis/programrule/engine/ProgramRuleEngine.java

```diff
@@ -79,55 +75,41 @@
     @NonNull
     private final SupplementaryDataProvider supplementaryDataProvider;
 
-    public List<RuleEffect> evaluate( ProgramInstance enrollment, Set<ProgramStageInstance> events )
+    public List<RuleEffect> evaluateOldOne( ProgramInstance enrollment, Set<ProgramStageInstance> events )
```

Comment:
The method names in this class are going to be changed in the next PR when I am going to refactor this class and the rule engine library part